### PR TITLE
Jackson ObjectMapper.readValue() method replacements

### PR DESCRIPTION
**Pull request by seran**
Skipped all the methods which uses `Jackson` classes as method arguments.

### CODE REVIEW

**Review by man-zhang — CHANGES_REQUESTED** [review]
hi, Seran, thanks. I did not find tests that use the method replacement you introduced for Jackson. please add them (see an example `JacksonEMTest`).

File: client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/thirdpartyclasses/JacksonObjectMapperClassReplacement.java [hunk 1]
```diff
@@ -35,65 +33,118 @@ protected String getNameOfThirdPartyTargetClass() {
...
         String content = new BufferedReader(
                 new InputStreamReader(src, Charset.defaultCharset()))
                 .lines()
                 .collect(Collectors.joining(System.lineSeparator()));
 
-        JsonTaint.handlePossibleJsonTaint(content,valueType);
+        ClassToSchema.registerSchemaIfNeeded(typeClass);
+        JsonTaint.handlePossibleJsonTaint(content, typeClass);
 
         src = new ByteArrayInputStream(content.getBytes());
```
Comment (arcuri82) [inline client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/thirdpartyclasses/JacksonObjectMapperClassReplacement.java:84]: there is quite a bit of code here that is the same as previous function. should be possibly refactored in a private method to avoid code duplication

File: scripts/loopback-aliases-macos.sh [hunk 2]
```diff
@@ -32,7 +32,7 @@ fi
 
 # Function to set aliases
 set_alias() {
-	for i in {2..254}
+	for i in {2..20}
```
Comment (arcuri82) [inline scripts/loopback-aliases-macos.sh:35]: why this change?

File: client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/thirdpartyclasses/JacksonObjectMapperClassReplacement.java (hunk 1 above)
Comment (arcuri82) [reply client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/thirdpartyclasses/JacksonObjectMapperClassReplacement.java:84]: but, to save time, i ll do this refactoring.

**Review by arcuri82 — CHANGES_REQUESTED** [review]

File: client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/thirdpartyclasses/JacksonObjectMapperClassReplacement.java (hunk 1 above)
Comment (seran) [reply client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/thirdpartyclasses/JacksonObjectMapperClassReplacement.java:85]: Thank you.

File: scripts/loopback-aliases-macos.sh (hunk 2 above)
Comment (seran) [reply scripts/loopback-aliases-macos.sh:35]: If I set till 254, internet stop working. I don't know why, so changed it for now.

Comment (arcuri82) [reply scripts/loopback-aliases-macos.sh:35]: in that case, add such comment
